tools/testing-utils/snapshot.tsx

```diff
@@ -91,6 +92,8 @@ export const snapshot = async (
         </body>
     </html>`;
 
+  await fs.writeFile('index.html', html, 'utf8');
```

Comment:
looks like you needed it for debug your case...
if we will save `html` locally, let's move it to some `tmp` folder, m?

Reply:
Yes, it was a debug statement, thank you for noticing